### PR TITLE
Fix ulimit issue with docker

### PR DESCRIPTION
**Pull request by Benkia**

### CODE REVIEW

File: skipper/runner.py [hunk 1]
```diff
@@ -60,6 +60,9 @@ def _run_nested(fqdn_image, environment, command, interactive, name, net, publis
     else:
         cmd += ['--rm']
 
+    if not os.environ.get("NO_ULIMIT", False):
+        cmd += ['--ulimit', "nofile=1024000:1024000"]
```
Comment (ofir-amir) [inline skipper/runner.py]: 1. why different?
2. maybe:
```suggestion
    DEFAULT_SKIPPER_ULIMIT = os.environ.get('SKIPPER_ULIMITS', 'nofile=65536:65536')
    ....
    ulimits = DEFAULT_SKIPPER_ULIMIT.split(',')  # format: x=y,a=b
    for limit in unlimits.split(','):
        cmd += ['--ulimit', limit]
```

File: skipper/cli.py [hunk 2]
```diff
@@ -407,7 +407,8 @@ def _prepare_build_container(registry, image, tag, git_revision, container_conte
     else:
         build_context = '.'
 
-    command = ['build', '--network=host', '-t', tagged_image_name, '-f', docker_file, build_context]
+    command = ['build', '--network=host', '-t', tagged_image_name, '-f', docker_file, build_context,
+               "--ulimit", "nofile=65536:65536"]
```
Comment (ofir-amir) [inline skipper/cli.py]:
```suggestion
    ulimits = DEFAULT_SKIPPER_ULIMIT.split(',')  # format: x=y,a=b
    for limit in unlimits.split(','):
        command += ['--ulimit', limit]
```

**Review by ofir-amir — APPROVED** [review]

File: skipper/runner.py (hunk 1 above)
Comment (eranco74) [reply skipper/runner.py]: @ofir-amir, why split(",") twice?
also if we support SKIPPER_ULIMIT in env we should support it with a flag as well.

File: skipper/cli.py (hunk 2 above)
Comment (eranco74) [reply skipper/cli.py]: Consider adding a function in utils that returns a list with the ulimit for the command and call it from both places.

**Review by ofir-amir — APPROVED** [review]

**Pull request merged**